packages/client/src/v2-events/features/events/registered-fields/Address.tsx

```diff
@@ -161,13 +163,25 @@ const URBAN_FIELDS = [
   }
 ] as const satisfies FieldConfigWithoutAddress[]
 
+function isFarajaland() {
+  return createFieldCondition('country').isEqualTo('FAR')
+}
+
+function otherThanFarajaland() {
+  return and(
+    not(createFieldCondition('country').isUndefined()),
+    not(isFarajaland())
+  )
+}
```

Comment:
The ticket might not make this very clear but I think the distinction between the fields is not Farajaland vs other countries but rather `window.config.COUNTRY` vs other countries. So domestic address vs international address.